src/components/v-mapbox-layer.js

```diff
@@ -1,44 +1,77 @@
 export default {
   name: 'v-mapbox-layer',
-  render () { },
-  data () {
-    return {
-    };
-  },
+
+  inject: ['getMap'],
+
+  render: () => null,
+
   props: {
-    options: {
-      default: () => {
-        return {};
-      },
-      type: [Object, String]
+    layer: {
```

Comment:
I realize this is a 'breaking' change. I find naming the prop `layer` much more descriptive. Point of discussion. In any case - I have more proposals for 'breaking changes' upcoming, so if preferred we could also group them into a single release that bumps version number.

Reply:
I would suggest to not introduce breaking changes. You could consider breaking changes if they allow for a new path of functionality to open up. In this case you have a layer, which should be equivalent to the mapbox-gl [layer](https://docs.mapbox.com/mapbox-gl-js/style-spec/layers/).  In the docs of the layer they refer to the properties of a layer as `properties`. 

The type [Object, String] should be replaced by Object. That's a good ijmprovement anyway.